src/binary_converter.py

```diff
@@ -0,0 +1,17 @@
+# N = int(input("inserisci un numero: "))
+
+
+def convert(N):
+    MAX_BITS = 32
+
+    j = MAX_BITS - 1
+
+    bits = [0] * MAX_BITS
+
+    while N > 0:
+        bits[j] = N % 2
+        N = N // 2
+        j -= 1
+    return "".join([str(x) for x in bits])
+
+print(convert(10))
```

Comment:
Aggiungi un 
```python
if __name__ == "__main__":
    print(convert(10))
```
come [qui](https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do)

Reply:
fatto